### PR TITLE
Updated Lab2 per feedback post 10/13 dry-run

### PR DESCRIPTION
**Pull request by kraman-aws**
*Description of changes:*
- Major changes: Updated FSx creation and DRA using CLI
- Other changes per feedback

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/08-fsx-lustre/04-update-cluster.md [hunk 1]
```diff
@@ -50,33 +45,44 @@ aws s3 cp mount-fsx.sh s3://${BUCKET_NAME_DATA}
...
 export S3PATH=s3://${BUCKET_NAME_DATA}/mount-fsx.sh
 yq -i '(.Scheduling.SlurmQueues[0].CustomActions.OnNodeConfigured.Script=env(S3PATH)) |
-      '(.Scheduling.SlurmQueues[0].Iam.AdditionalIamPolicies[0]={"Policy": "arn:aws:iam::aws:policy/AmazonFSxFullAccess"}) |
-      '(.Scheduling.SlurmQueues[0].Iam.AdditionalIamPolicies[1]={"Policy": "arn:aws:iam::aws:policy/AmazonS3FullAccess"})' my-cluster-config.yaml
+       (.Scheduling.SlurmQueues[0].Iam.AdditionalIamPolicies[0]={"Policy": "arn:aws:iam::aws:policy/AmazonFSxFullAccess"}) |
+       (.Scheduling.SlurmQueues[0].Iam.AdditionalIamPolicies[1]={"Policy": "arn:aws:iam::aws:policy/AmazonS3FullAccess"})' ~/environment/my-cluster-config.yaml
 ```
 
-6. Update the cluster
+6. Stop the compute fleet before updating the cluster. Note that the compute fleet needs to be stopped even if no compute instances are running.
```
Comment (mhuguesaws) [inline content/08-fsx-lustre/04-update-cluster.md:60]: This needs to be done at the end of lab1

**Review by mhuguesaws — COMMENTED** [review]
Left some comments.

File: content/08-fsx-lustre/04-update-cluster.md [hunk 2]
```diff
@@ -50,33 +45,44 @@ aws s3 cp mount-fsx.sh s3://${BUCKET_NAME_DATA}
 VERSION=v4.28.1
```
[the remark this answers is not in the document]
Comment (kraman-aws) [reply content/08-fsx-lustre/04-update-cluster.md:45]: addressed in this: https://github.com/aws-samples/aws-hpc-tutorials/pull/336

File: content/08-fsx-lustre/04-update-cluster.md (hunk 1 above)
Comment (kraman-aws) [reply content/08-fsx-lustre/04-update-cluster.md:60]: addressed in this: https://github.com/aws-samples/aws-hpc-tutorials/pull/336

File: content/08-fsx-lustre/04-update-cluster.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mhuguesaws) [reply content/08-fsx-lustre/04-update-cluster.md:45]: Please remove the section from the PR @kraman-aws

[the remark this answers is not in the document]
Comment (mhuguesaws) [reply content/08-fsx-lustre/04-update-cluster.md:45]: I saw into the other PR

**Review by mhuguesaws — APPROVED** [review]
LGTM

**Pull request merged**